src/index.html

```diff
@@ -1,17 +1,154 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="form.css">
+    <title>Document</title>
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <header>
+
+    </header>
+
+    <main>
+      <form
+        action=
+        "https://mate-academy-form-lesson.herokuapp.com/api"
+        method="post"
+      >
+        <fieldset class="fieldset">
+
+          <legend>Personal information</legend>
+
+          <label class="label">
+            Surname:
+            <input type="text" name="surname">
+          </label>
+
+          <label class="label">
+            Name:
+            <input type="text" name="name" required>
+          </label>
+
+          <label class="label">
+            How old are You?
+            <input
+              type="text"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+              required
+            >
+          </label>
+
+          <label class="label">
+            Full date of birth:
+            <input type="date" name="birthDate">
+          </label>
+
+          <label class="label">
+            I accept the term of the agreement
+            <input type="checkbox" name="term" required>
+          </label>
+        </fieldset>
+
+        <fieldset class="fieldset">
+
+          <legend>Registration</legend>
+
+          <label class="label">
+            E-mail:
+            <input type="text"
+                  name="email"
+                  placeholder="email@example.com"
+                  required
+            >
```

Comment:
fix indents in all places
```suggestion
            <input
              type="text"
              name="email"
              placeholder="email@example.com"
              required
            >
```